clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/NativeProtocolResourceDefinitionRegistrar.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.jboss.as.clustering.jgroups.subsystem;
+
+import org.jboss.as.controller.PathElement;
+import org.jgroups.Global;
+import org.jgroups.stack.Protocol;
+import org.wildfly.subsystem.resource.operation.ResourceOperationRuntimeHandler;
+
+/**
```

Comment:
Some description of purpose would improve readability.

Reply:
Fixed.